go/service/chat_local_test.go

```diff
@@ -152,12 +153,13 @@ func TestGetThreadLocal(t *testing.T) {
 	if len(tv.Messages) != 2 {
 		t.Fatalf("unexpected response from GetThreadLocal . expected 2 items, got %d\n", len(tv.Messages))
 	}
-	if tv.Messages[0].MessagePlaintext.MessageBodies[0].Text().Body != "hello!" {
+	if tv.Messages[0].MessagePlaintext.V1().MessageBody.Text().Body != "hello!" {
 		t.Fatalf("unexpected response from GetThreadLocal . expected 'hello!' got %#+v\n", tv.Messages[0])
 	}
 }
 
 func TestGetInboxSummaryLocal(t *testing.T) {
+	t.Skip("this needs to be fixed")
```

Comment:
We talked about it in slack#chat, song is going to look into fixing these tests.